osbs/build/build_request.py

```diff
@@ -136,6 +137,20 @@ def inner_template(self):
         return self._inner_template
 
     @property
+    def customize_conf(self):
+        if self._customize_conf is None:
+            path = os.path.join(self.build_json_store, DEFAULT_CUSTOMIZE_CONF)
+            logger.debug("loading customize conf from path %s", path)
+            try:
+                with open(path, "r") as fp:
+                    self._customize_conf= json.load(fp)
+            except IOError:
+                # File not found, which is perfectly fine. Set to empty string
+                self._customize_conf = ""
```

Comment:
Can this default to an empty `dict` instead? It should only ever be `None` or a `dict`.